From c6e3c512c9a132bde97917ad15abe1da184efdd2 Mon Sep 17 00:00:00 2001
From: Charles Stevens <cstevens@beaconembedded.com>
Date: Thu, 20 Jan 2022 11:45:17 -0600
Subject: [PATCH 67/68] [RSOM-360] - dts: beacon: add missing usb phy ref

Add missing reference to the [optional] usb phy for the USB3
peripheral controller.

Signed-off-by: Charles Stevens <cstevens@beaconembedded.com>
---
 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
index fa9aa705bf23..1232c356fa73 100644
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
@@ -876,6 +876,8 @@
 	companion = <&xhci0>;
 	status = "okay";
 	usb-role-switch;
+        phys = <&usb3_phy0>;
+        phy-names = "usb";
 
 	port {
 		usb3_role_switch: endpoint {
-- 
2.17.1

